bin/deployment/ProjectInitializer.php

```diff
@@ -77,6 +79,40 @@ public function setAuthor($name, $email)
         return $this;
     }
 
+    /**
+     * @param string $preprocessor
+     * @return $this
+     */
+    public function setCssPreprocessor($preprocessor) {
+        switch($preprocessor) {
+            case 'sass':
+                $this->bootstrap = ['name' => 'bootstrap-sass', 'version' => '3.3.7'];
+                $this->removeDirecory($this->dir . '/frontend/src/less');
```

Comment:
The actual work should be done in initialize() method. This setter should only prepare configuration in ProjectInitializer attributes.
Please add SASS/LESS choice to initializeFromConsole() method too.